clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java

```diff
@@ -242,6 +242,11 @@
     private static final String ALLOW_AUTO_CREATE_TOPICS_DOC = "The client-side (producer) permission to allow auto-topic creation. Both the client-side and the broker-side should enable auto-topic creation in order for a topic to be automatically created";
     public static final boolean DEFAULT_ALLOW_AUTO_CREATE_TOPICS = false;
 
+    /** <code>disable.passthrough.batching</code> */
+    public static final String DISABLE_PASSTHROUGH_BATCHING_CONFIG = "disable.passthrough.batching";
```

Comment:
Would it be better to name it as "enable..." instead of "disable..."?
It might be easier to understand like if you set this to true, batching is enabled and if set to false, batching is disabled.

Reply:
good suggestion! Updated